sequencer/src/state.rs

```diff
@@ -51,8 +65,19 @@ impl hotshot_types::traits::state::TestableState for State {
     }
 }
 
-impl Committable for State {
+impl Committable for ValidatedState {
```

Comment:
I don't think this impl is, or should be, required anymore. Since we really only deal with the header commitment.

If we are going to implement it, we should commit to the roots, not the entire trees.

Reply:
It is currently a bound on the trait: https://github.com/EspressoSystems/HotShot/blob/160e8c52e63a5bacb94751fa6777d1703b94986d/crates/types/src/traits/state.rs#L38

Reply:
We should be able to remove this now

Reply:
Whoops commented on the wrong review